tests/rust-integration-tests/integration_test/src/tests/hostname/mod.rs

```diff
@@ -0,0 +1,53 @@
+use anyhow::{Context, Result};
+use oci_spec::runtime::{LinuxBuilder, ProcessBuilder, Spec, SpecBuilder};
+use test_framework::{Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+
+fn create_spec(hostname: &str) -> Result<Spec> {
+    Ok(SpecBuilder::default()
+        .hostname(hostname)
+        .linux(
+            LinuxBuilder::default()
+                .readonly_paths(vec![
+                    "/proc/bus".to_string(),
+                    "/proc/fs".to_string(),
+                    "/proc/sys".to_string(),
+                ])
+                .build()
+                .expect("should build linux config"),
+        )
+        .process(
+            ProcessBuilder::default()
+                .args(vec!["runtimetest".to_string()])
+                .build()
+                .expect("can create process"),
+        )
+        .build()
+        .context("Failed to build hostname spec")?)
```

Comment:
```suggestion
        .context("failed to build hostname spec")?)
```
The error messages should always be lowercase :sweat_smile: 
Also, we can just unwrap here, as failing this should fail the case anyways ( and realistically this should never fail)